.gitignore

```diff
@@ -20,3 +20,4 @@ sagelib
 *.pyc
 *.pyo
 __pycache__
+*.egg-info
```

Comment:
@divergentdave please confirm we don't want to check this in.

Reply:
Yes, LGTM.